sfcasts/webhook-consumer.md

```diff
@@ -1,149 +1,81 @@
-add a product to the cart, and proceed to checkout. Oops,
-another error. 
-
-If we debug the response content to check the actual error in `LemonSqueezyApi`,
-we'll see that the field must be a string and it points to our custom
-`user_id` we added. Note that you can't pass just *anything*, so passing the
-whole user object definitely won't work. 
-
-Alright, let's explicitly typecast user ID to a string and try again.
-Success! We're on the checkout page. I'll quickly fill in some data, make
-the payment, and wait for the webhook. And yes, this one was accepted now
-with the 202 status code.
-
-Looking at the request data, we see our custom data with `user_id=1`.
-Checking the database, we can quickly type an SQL command in my console
-`bin/console doctrine:query:sql` and in double quotes input
-`SELECT * FROM user WHERE id = 1` as we have the ID = 1 for the current user:
-
-```terminal-silent
-bin/console doctrine:query:sql "SELECT * FROM user WHERE id = 1"
+Time to test the webhook again! I'm a fan of the Ngrok inspector, so I'll use that. Hmm... an *error*. The webhook doesn't have a `user_id` set on custom data for that specific case. That makes sense. With Ngrok, we can modify the original webhook content and *replay* it with modifications. But to be *extra* sure our changes are accounted for, we'll go through the checkout process again so LemonSqueezy can set the `user_id` correctly.
+
+Let's log in again, add a product to the cart, and try to check out. Oops, *another* error. Let's head back to our code and uncomment this `dd()` to get a better picture of what's happening here. If we refresh our site... ah! It looks like the field needs to be a string, and it's pointing to the custom `user_id` we added. You can't pass just *anything*, so passing the whole user object definitely won't work. Head back to our code and comment out that `dd()` again... then, up here... let's specify that this user ID is a `string` and try this again. When we refresh... success! We're on the checkout page!
+
+Let's fill in the card info... address... make the payment, and wait for the webhook. *Yes*! Our transaction was accepted and, over here, we have a 202 status code.
+
+If we look at the request, we can see our `custom_data` where the `user_id` equals `1`. We can also check the database with a handy dandy SQL command. At your terminal, run
+
+```terminal
+bin/console doctrine:query:sql
+```
+
+and in double quotes, input
+
+```terminal
+SELECT * FROM user WHERE id = 1
 ```
 
-Hit enter and yes, `lsCustomerId` is set to this ID. We can
-now leverage this in our application, for instance, to show the link to
-order lists made by this customer. But before we do that, let's see how we
-can test our webhook. That's coming up next!
+since we have an ID of "1" for the current user. Hit "enter" and... *yes*! The `lsCustomerId` is set to this unique ID. This is what we need to create a list of orders made by this customer.
```

Comment:
```suggestion
since we have an ID of "1" for the current user. Hit `Enter` and... *yes*! The `lsCustomerId` is set to this unique ID. This is what we need to create a list of orders made by this customer.
```